string player_id
string color_str
---
GameState game_state
